linux/shim/io.go

```diff
@@ -15,8 +15,9 @@ import (
 )
 
 func copyConsole(ctx context.Context, console console.Console, stdin, stdout, stderr string, wg, cwg *sync.WaitGroup) error {
```

Comment:
actually one more nit. Either pass `context.Background()` as `ctx` here, or remove it :sweat_smile: